### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Zedb0T**

### CODE REVIEW

File: Launcher with autoupdater.py [hunk 1]
```diff
@@ -78,21 +39,18 @@ def try_remove_dir(dir):
...
 def downloadNewestmod():
-	print("test")
+	
 	InstallDir = AppdataPATH
 	
-	launchUrl ="https://api.github.com/repos/OpenGOAL-Unofficial-Mods/OpenGoal-ModLauncher-dev/releases"
-	PARAMS = {'address':"yolo"} 
-	r = json.loads(json.dumps(requests.get(url = launchUrl, params = PARAMS).json()))
+	launchUrl ="https://api.github.com/repos/OpenGOAL-Unofficial-Mods/OpenGoal-ModLauncher-dev/releases"  
+	r = json.loads(json.dumps(requests.get(url = launchUrl, params = {'address':"yolo"}).json()))
```
Comment (dallmeyer) [inline Launcher with autoupdater.py:46]: do we even need the params? I think I tried once w/o and it worked the same but never checked it in

File: Launcher with autoupdater.py [hunk 2]
```diff
@@ -138,26 +96,17 @@ def downloadNewestmod():
...
 
 if os.path.exists(AppdataPATH) == False:
 	print("Creating Directory " + AppdataPATH)
 	os.mkdir(AppdataPATH)
 
-	if os.path.exists(AppdataPATH + "\\OpengoalModLauncher.exe") == False:
-		print("No Mod Launcher detected")
-		downloadNewestmod()
-print("test")
+if os.path.exists(AppdataPATH + "\\OpengoalModLauncher.exe") == True:
```
Comment (dallmeyer) [inline Launcher with autoupdater.py:106]: ? leftover from testing maybe?

**Review by dallmeyer — COMMENTED** [review]
otherwise looks fine to me

File: Launcher with autoupdater.py (hunk 1 above)
Comment (Zedb0T) [reply Launcher with autoupdater.py:46]: It didnt work when I tested If I recall correctly
I can test if I remeber tho

File: Launcher with autoupdater.py (hunk 2 above)
Comment (Zedb0T) [reply Launcher with autoupdater.py]: Fixed

**Pull request merged**